components/IscnInfoCard.vue

```diff
@@ -109,6 +109,28 @@
             </template>
           </Button>
         </div>
+        <div
+          :class="[
+            'p-[8px]',
+            'rounded-[20px]',
+            'my-[12px]'
+          ]"
+        >
+          <Button
+            preset="secondary"
+            :text="$t('HomePage.mint.button')"
+            :to="localeLocation({ name: 'nft' })"
+            class="text-like-green"
+            :style="{ backgroundColor: '#AAF1E7', color: '#28646E' }"
+          >
+            <template #prepend>
+              <IconMint class="w-[20px]" />
```

Comment:
Please use this icon
![mint-with-hex](https://user-images.githubusercontent.com/17264716/216219914-6990b5b8-b471-4770-872f-30a61db7db53.svg)

Reply:
@williamchong Updated icon
![mint-with-hex-updated](https://user-images.githubusercontent.com/17264716/216262794-5eaed3cf-88ab-4c5e-928a-1031ad134ba2.svg)

Reply:
![Uploading image.png…]()